app/metrics.yaml

```diff
@@ -5394,3 +5394,53 @@ set_default_setting_experiment:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2021-10-01"
+
+credit_cards:
+  manual_save:
+    type: counter
+    description: |
+      A counter of the number of credit cards that have been saved
+      manually by the user.
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/18711
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/19548
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2021-10-01"
```

Comment:
nit: the expiry date might be too close?

Reply:
It can be easily extended later if need-be.  The data-review form for extending a collection is even shorter to help facilitate this.